constants.xml

```diff
@@ -190,7 +190,7 @@
         <CONNECTION_ALL_TIMEOUT>60</CONNECTION_ALL_TIMEOUT>
         <!-- Timeout in seconds for ONLY connection that reach our callback function, 0 means no timeout-->
         <CONNECTION_CALLBACK_TIMEOUT>0</CONNECTION_CALLBACK_TIMEOUT>
-        <ENABLE_EVM>true</ENABLE_EVM>
+        <ENABLE_EVM>false</ENABLE_EVM>
```

Comment:
cc @valeryz @its-saeed Do we need to update any configuration in CI if we're changing this default to make sure the tests still pass?